internal/impl/kafka/output_schema_registry.go

```diff
@@ -23,10 +23,10 @@ import (
 	"net/http"
 	"net/url"
 	"slices"
+	"strings"
 	"sync"
 	"sync/atomic"
 
-	"cuelang.org/go/pkg/strings"
```

Comment:
My bad, I didn't notice when this import got added automatically in a previous PR...